apparmor.d/abstractions/app/chromium

```diff
@@ -26,6 +26,7 @@
   include <abstractions/fontconfig-cache-read>
   include <abstractions/graphics-full>
   include <abstractions/nameservice-strict>
+  include <abstractions/qt5>
```

Comment:
`abstractions/desktop` already includes `qt5` (it is a agnostic version of `kde-strict`). So you don't need it here. Same elsewhere.

Reply:
Fixed